bom-weekly/pom.xml

```diff
@@ -253,7 +263,17 @@
             <dependency>
                 <groupId>org.jenkins-ci.plugins</groupId>
                 <artifactId>gitlab-oauth</artifactId>
-                <version>1.15</version>
+                <version>1.16</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>gitlab-plugin</artifactId>
+                <version>1.5.31</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>handy-uri-templates-2-api</artifactId>
+                <version>2.1.8-2.0-rc21.df08d16e72dd</version> <!-- TODO https://github.com/jenkinsci/handy-uri-templates-2-api-plugin/pull/3 -->
```

Comment:
https://github.com/jenkinsci/handy-uri-templates-2-api-plugin/pull/3

@jetersen I see you have access in RPU, would you mind releasing this?

Reply:
```suggestion
                <version>2.1.8-22.v77d5b_75e6953</version>
```